### ISSUE TITLE
Add .inline class for icons known to exist predominantly inline

### ISSUE THREAD

**Issue by scott-joe** [ExactTarget/fuelux-mctheme#339]
For whatever reason, the `.fueluxicon-question-mark` icon isn't a part of the icon `.fuelux-icon` library, but it's own special class. Which isn't a big deal, just thought I'd mention it as well. But it gave me the idea for a class to offset of some sort to vertically center them with the line height instead of the text's baseline location... for this particular use, I simply offset by 2px but it'd be more complex than that to account for many font sizes...I haven't played with the icons enough to know how many of these are used inline with text.

`.fueluxicon-question-mark`
<img width="142" alt="screen shot 2015-08-05 at 4 35 40 pm" src="https://cloud.githubusercontent.com/assets/2026866/9139960/9c48a678-3cfb-11e5-9ac2-377093d2690b.png">

`.fueluxicon-question-mark.icon-inline { position: relative; top: 2px; }`
<img width="148" alt="screen shot 2015-08-05 at 4 35 46 pm" src="https://cloud.githubusercontent.com/assets/2026866/9139963/9f3db9fe-3cfb-11e5-92ad-4f3d02fa0447.png">

**Comment by interactivellama** [comment]
Yes that icon was an ask before we got the awesome SVG stuff going. Probably needs "alignment" --pun intended!

**Comment by scott-joe** [comment]
lol

*Scott Williams*

*Sr. Front-End Engineer, UX Architecture*UXDS, Salesforce Marketing Cloud
